### ISSUE TITLE
下载看不到进度

### ISSUE THREAD

**Issue by changqk** [Eugeny/tabby#8139]
<img width="346" alt="image" src="https://user-images.githubusercontent.com/16031023/227471916-1cc2f63d-6cee-4858-9bdb-ee20d25c83ef.png">
下载看不到进度

**Comment by changqk** [comment]
说错了，是上传看不到进度

**Comment by JaminYe** [comment]
Yes, it happened to me too. I can’t use the latest version 196. I return to version 185 and it works. My device is apple m1 air, and the system version is 12.6.3

**Comment by TaiJi2022** [comment]
Yes, I have the same problem, I can not see the progress when uploading the file
